app/classifier/index.cjsx

```diff
@@ -655,6 +672,8 @@ module.exports = React.createClass
       <Classifier {...@props}
         workflow={@props.workflow}
         subject={@state.subject}
+        guide={@state.guide}
```

Comment:
I don't think the classifier itself needs the field guide and its linked media as props?